src/main/resources/templates/index.html

```diff
@@ -13,16 +13,16 @@
 
     <ul class="nav">
         <li><a href="index.html">Search</a></li>
-        <li><a href="AddFacility.html">Add Faculty</a></li>
+        <li><a href="addFacility.html">Add Faculty</a></li>
     </ul>
 
-    <form action="#" th:action="${/facility}" method="GET">
+    <form action="#" th:action="@{/facility}" method="GET">
```

Comment:
This EL expression was invalid and caused the following exception, which prevents the Index.html page from loading.  

THYMELEAF][http-nio-8080-exec-2] Exception processing template "index": Exception evaluating SpringEL expression: "/facility" (template: "index" - line 19, col 22)

org.thymeleaf.exceptions.TemplateProcessingException: Exception evaluating SpringEL expression: "/facility" (template: "index" - line 19, col 22)

I would recommend changing this to prevent the exception above by using the @ instead of /